spec/lib/models/locales_updater_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'spec_helper'
+require_relative '../../../lib/open_project/translations/models/locales_updater'
+
+describe LocalesUpdater do
+  let(:locales_updater) { LocalesUpdater.new }
+  let(:configuration) {
+    { plugins:
+      { 'test-project' =>
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.